### PR TITLE
docs(readme): optimize readmes

### PR DESCRIPTION
**Pull request by jiazhewang**
- Move language switches up to the top.
- Center the title, subtitle and badges.
- Center the images.
- Optimize the lines in the English version README to make them simple and neat.
- Remove the Development sections as they only need to be in the contributing guides.
- Add Links, IMPORTANT!

TODO:
- Add the Development (or Build) section back to the contributing guides.
- Check the bar chart example: NOT consistent with the demo on the offical site (the legend part).
- Change every content (e.g. '1951' not '1951 年') of the example in English in the English version README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,24 +86,20 @@ const bar = new Bar(document.getElementById('container'), {
...
+Your contributions are always welcome! Please Do have a look at the [issues](https://github.com/antvis/g2plot/issues) first.
 
-# run test case
-$ npm run test
+To become a contributor, please follow our [contributing guide](https://github.com/antvis/g2plot/blob/master/CONTRIBUTING.md).
 
-# run demos
-$ npm start
-```
+## License
```
Comment (hustcc) [inline README.md]: 一般，License 都是放到最后一个 header 吧~~

Comment (jiazhewang) [reply README.md]: Accroding to some awesome-readme & guide ([example](https://gist.github.com/PurpleBooth/109311bb0361f32d87a2)): Acknowledgments, links,... appear later than License.

File: README.md [hunk 2]
```diff
@@ -1,40 +1,53 @@
...
 
+<div align="center">
 <img src="https://gw.alipayobjects.com/mdn/rms_d314dd/afts/img/A*rqI2Qqt0pTwAAAAAAAAAAABkARQnAQ" width="600" />
+</div>
 
-### responsive charts
+### Responsive
 
-The responsive ability of G2Plot guarantee your chart’s readability and information density in any display space and data status.
+G2Plot guarantees the readability of the charts in different sizes.
```
Comment (paleface001) [inline README.md]: 感觉这句不妥，实际上响应式不光跟显示尺寸有关，也跟数据状态有关

File: README.md [hunk 3]
```diff
@@ -1,40 +1,53 @@
...
+<img src="https://gw.alipayobjects.com/mdn/rms_d314dd/afts/img/A*gV_JQZVbDWAAAAAAAAAAAABkARQnAQ" width="200">
+</div>
+<br/>
 
 ## Features
 
-### pretty and high-quality charts out of box
-
-G2Plot is powered by an enterprise-class visualization design principle, which not only focuses on the entire visual style of the charting library, but also provides users the best default options for each chart concerning to their own features.
+### Pretty & Lightweight
```
Comment (paleface001) [inline README.md]: out of box 挺好的呀

Comment (jiazhewang) [reply README.md]: > out of THE box

But it is not really 'out-of-the-box'...

The out-of-the-box feature or functionality in software is something that works immediately **WITHOUT ANY configuration or customizations**. In fact, G2Plot is selling its SIMPLE configurations simultaneously.

> https://www.urbandictionary.com/define.php?term=out-of-the-box
> https://en.wikipedia.org/wiki/Out_of_the_box_(feature)

File: README.md (hunk 2 above)
Comment (jiazhewang) [reply README.md]: Sure, could be:

> G2Plot guarantees the readability of charts in different sizes and data.

**Pull request merged**